### PR TITLE
Initialize schedulers when the DAG FSM is restarted.

### PR DESCRIPTION
**Pull request by pribor**

### CODE REVIEW

File: core/src/main/scala/gwi/s8/DagFSM.scala [hunk 1]
```diff
@@ -115,11 +115,18 @@ class DagFSM(
...
+  }
+
   def applyEvent(event: DagStateEvent, dagState: DagFSMState): DagFSMState = event match {
     case e =>
       dagState.updated(e)
   }
 
+  override def onRecoveryCompleted() {
+    cancellables = schedulePartitionCheck(stateData) ++ cancellables
+  }
```
Comment (l15k4) [inline core/src/main/scala/gwi/s8/DagFSM.scala]: @pribor I don't understand this, shouldn't `schedulePartitionCheck` be called also on initialization, not just on recavery?

Comment (pribor) [reply core/src/main/scala/gwi/s8/DagFSM.scala]: In the case of PersistentFSM it is called always even if there is no previous state/events, check tests..

Comment (pribor) [reply core/src/main/scala/gwi/s8/DagFSM.scala]: https://doc.akka.io/docs/akka/2.5.4/scala/persistence.html#recovery-status

Comment (l15k4) [reply core/src/main/scala/gwi/s8/DagFSM.scala]: Nice one, this would save me a lot of troubles,I was struggling with this problem a lot. Nice work !

**Pull request merged**